tests/modules/list.c

```diff
@@ -213,6 +214,32 @@ int list_delete(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     return REDISMODULE_OK;
 }
 
+/* this flag is used to work with busy commands, that might take a while
+ * and ability to stop the busy work with a different command*/
+static int abort_flag = 0;
+
+int list_busy(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
```

Comment:
it's a little bit odd to see it in the "list" test module.
maybe putting it in the timer module is less awkward?